src/SolutionValidator/ProjectLoader.cs

```diff
@@ -50,12 +50,7 @@ public ProjectLoader(ILogger<ProjectLoader> logger)
 
             if (sdkVersion >= new Version(6, 0))
             {
-#if NET6_0_OR_GREATER
-                Environment.SetEnvironmentVariable("MSBUILDADDITIONALSDKRESOLVERSFOLDER", Path.Combine(sdkDirectory, "SdkResolvers"));
-                Environment.SetEnvironmentVariable("MSBuildEnableWorkloadResolver", "true");
-#else
                 Environment.SetEnvironmentVariable("MSBuildEnableWorkloadResolver", "false");
-#endif
```

Comment:
Any thoughts on this @wo-roberts? It fixes the tests on Windows but I'm not quite sure this is what is expected.